### ISSUE TITLE
About the result of 14Lap

### ISSUE THREAD

**Issue by ValkyriaLenneth** [fengyh3/TSMSA#1]
Dear authors:
I run your code on my own server with dataset for TSMSA_bert_model:  14lap and the command `python TSMSA.py --do_train=true --do_predict=true --data_dir=./data_TOWE/19data/14lap --vocab_file=./bert_base/vocab.txt --bert_config_file=./bert_base/bert_config.json --init_checkpoint=./bert_base/bert_model.ckpt --max_seq_length=100 --train_batch_size=32 --learning_rate=5e-5 --num_train_epochs=8.0 --output_dir=./output` 

However, the result of evaluation is `precision: 0.8003731343283582, recall: 0.7566137566137566, f1: 0.777878513096005` which is far lower than the results in the paper. 

Could you give me some advice on the reproduction of the result?

**Comment by fengyh3** [comment]
May be you can try to delete the random seed of tf in TSMSA. It causes different results in different device. This is my running result in 14Lap today: precision: 0.8366972477064221, recall: 0.8042328042328042, f1: 0.8201438848421059
Moreover, due to the different dataset, the hyper-paramaters are not the same, but similar: num_train_epochs are 2 to 8 and learning_rates are 2e-5 or 5e-5.

**Comment by fengyh3** [comment]
supplement in 14res：precision: 0.8786359077231695, recall: 0.8504854368932039, f1: 0.8643315243703388
learning_rate: 5e-5, epoch: 6

**Comment by ValkyriaLenneth** [comment]
Thanks for reply, could you provide a full configuration and seed of all four TOEW datasets?

**Comment by ValkyriaLenneth** [comment]
By the way, could you tell me your device, maybe I could find a same one.

**Comment by fengyh3** [comment]
I just delete the seed and the results are stable running for multiple times and different devices.

config: python TSMSA.py --do_train=true --do_predict=true --data_dir=./bert_model/data_TOWE/19data/15res --vocab_file=./bert_model/bert_base/vocab.txt --bert_config_file=./bert_model/bert_base/bert_config.json --init_checkpoint=./bert_model/bert_base/bert_model.ckpt --max_seq_length=100 --train_batch_size=32 --learning_rate=5e-5 --num_train_epochs=? --output_dir=./bert_model/output
* ? can try 2 to 8, mostly 3 or 4

for example, today i just run the 15res and get the result:precision: 0.8082191780821918, recall: 0.8377281947261663, f1: 0.8227091632966297.   epoch is 4.

device: the results of paper are running on win10 PC, and above results are running on my mac pro 19.  I think you can just delete the seed.

**Comment by ValkyriaLenneth** [comment]
Thanks very much. 
I used your config run the model on 15res and 14lap and delete the seed. The result is as the same level with yours. Maybe I would run the model for several times and have an average as the final result.
